### ISSUE TITLE
New Item: Pellet Life Regenerator

### ISSUE THREAD

**Issue by hop193** [BEEmod/BEE2-items#1817]
Simple. Has the properties of the fizzler, and when a Pellet passes through, the life of the pellet regenerated. If this isn't added, can you add "Auto-Respawn Cube" property to the Flux Feild to do so, as well as add something like "Start Enabled" to make it activate something when passed? Thanks!

**Comment by Konclan** [comment]
This would actually be really useful

**Comment by TeamSpen210** [comment]
I'm not sure if this is possible.

**Comment by Luke18033** [comment]
There isn't some sort of `ResetLife` or `SetLife` input?

**Comment by TeamSpen210** [comment]
No such luck.

**Comment by Luke18033** [comment]
How do you even configure the pellets' life? Is that what Respawn Time is?

**Comment by hop193** [comment]
I do know the life is, in fact, reset when going through a portal.

**Comment by Endermage77** [comment]
I wonder if the pellet renewal works with any worldportals or only with portals.
Because if it does, the pellet refresher could be instead a 1-voxel block with a worldportal hole going through it. And maybe some other decorations on it too.

**Comment by hop193** [comment]
So is this just abandoned?

**Comment by Endermage77** [comment]
Nah. Just wait a few weeks when somebody rediscovers the post and says "Oh! That's a thing!" and comments on it.

**Comment by TheLoafLord** [comment]
i like this idea alot.

**Comment by Endermage77** [comment]
...Bingo.
